.vib/dokuwiki/cypress/cypress/integration/dokuwiki_spec.js

```diff
@@ -1,52 +1,18 @@
 /// <reference types="cypress" />
 import { random } from '../support/utils';
 
-it('allows login and logout', () => {
-  cy.login();
-  cy.get('.error').should('not.exist');
-  cy.contains('Log Out').click();
-  cy.get('#login');
-});
-
-it('allows the user to update profile', () => {
-  cy.login();
-  cy.visit('/start?do=profile');
-  cy.fixture('admins').then((admin) => {
-    cy.get('input[name="email"]')
-      .clear()
-      .type(`${admin.newAdmin.email}.${random}`);
-  });
-  cy.get('input[name="oldpass"]').first().type(Cypress.env('password'));
-  cy.contains('Save').click();
-  cy.contains('User profile successfully updated');
-});
-
-it('allows the admin to validate and modify configuration settings', () => {
-  cy.login();
-  cy.visit('start?do=admin&page=config');
-  cy.fixture('settings').then((setting) => {
-    cy.get('#config___savedir').should(
-      'contain.value',
-      setting.newSetting.saveFolder
-    );
-    cy.get('#config___dmode').should(
-      'contain.value',
-      setting.newSetting.directoryCreationMode
-    );
-    cy.get('#config___title')
-      .scrollIntoView()
-      .clear()
-      .type(setting.newSetting.newDokuWikiFullName);
+it('allows editing the starting page', () => {
```

Comment:
IMHO, with the login and with this we are testing the same integration: database. I'd maybe complete this test updating an image to test the volume integration too.

Reply:
As discussed, I have reduced the number of tests to only one (plus login) as this particular chart does not use a DDBB, storing all data in its PVC. That means that as long as we are checking something like login or page edition we are also checking the PVC was mounted without issues.